sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/repository/integration/ContactRepositoryIT.java

```diff
@@ -33,7 +35,11 @@
 @ContextConfiguration(classes = TestRepositoryConfig.class)
 public class ContactRepositoryIT {
 
-    private static final Contact TEST_CONTACT = new Contact("testId", "faketitle");
+    private static final Contact TEST_CONTACT = new Contact("testId", "faketitle", 25, true);
+    private static final Contact TEST_CONTACT2 = new Contact("testId2", "faketitle2",  32, false);
+    private static final Contact TEST_CONTACT3 = new Contact("testId3", "faketitle3", 25, false);
+    private static final Contact TEST_CONTACT4 = new Contact("testId4", "faketitle4", 43, true);
+    private static final Contact TEST_CONTACT5 = new Contact("testId5", "faketitle3", 43, true);
```

Comment:
To make test code easier to read, could you please:
1. Delete `Contact.Contact()` and `Contact.Contact(String logicId, String title)`, only keep `Contact(final String logicId, final String title, final int intValue, boolean status)`.
2. Delete all contects create in unit tests, define them as static field, just like `TEST_CONTACT1` to `TEST_CONTACT5`

Reply:
Again, wan to keep the changes minimal and relevant to query annotation in this PR. I can clean the other tests in another PR

Reply:
OK, Thank you. Please create an issue, and put a link here.

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/14828